admin_guide/managing_networking.adoc

```diff
@@ -215,20 +209,16 @@ To configure egress policy:
 ----
 +
 When the example above is added in a project, it allows traffic to IP range
-`1.2.3.0/24` and domain name `www.foo.com`, but denies access to all other
-external IP addresses. (Traffic to other pods is not affected because the policy
-only applies to _external_ traffic.)
+`1.2.3.0/24`, but denies access to all other external IP addresses. (Traffic to
+other pods is not affected because the policy only applies to _external_
+traffic.)
 +
 The rules in an `EgressNetworkPolicy` are checked in order, and the first one
-that matches takes effect. If the three rules in the above example were
-reversed, then traffic would not be allowed to `1.2.3.0/24` and `www.foo.com`
-because the `0.0.0.0/0` rule would be checked first, and it would match and deny
-all traffic.
+that matches takes effect. If the two rules in the above example were reversed,
+then traffic would not be allowed to `1.2.3.0/24` because the `0.0.0.0/0` rule
+would be checked first, and it would match and deny all traffic.
 +
-Domain name updates are reflected within 30 minutes. In the above example,
-suppose `www.foo.com` resolved to `10.11.12.13`, but later it was changed to
-`20.21.22.23`. Then, {product-title} will take up to 30 minutes to adapt to
-these DNS updates.
+Domain name updates are reflected within 30 minutes. This includes if the values change.
```

Comment:
How about:
 - `Domain name updates (including IP address change) are reflected within 30 minutes.` OR 
 - `Domain names and IP address changes are reflected within 30 minutes.`